src/transformers/convert_slow_tokenizer.py

```diff
@@ -772,6 +772,90 @@ def post_processor(self):
         )
 
 
+class M2M100Converter(SpmConverter):
+    def __init__(self, original_tokenizer):
+        self.original_vocab_file = original_tokenizer.vocab_file
+
+        original_tokenizer.vocab_file = self.original_vocab_file.replace("vocab.json", "sentencepiece.bpe.model")
+        super().__init__(original_tokenizer)
+        original_tokenizer.vocab_file = self.original_vocab_file
+
+    def vocab(self, proto=None):
+        import json
+
+        with open(self.original_vocab_file) as fp:
+            vocab = json.load(fp)
+
+        vocab = list(sorted(vocab.items(), key=lambda x: x[1]))
```

Comment:
If `<s>` and `<unk>` etc are not part of the `original_vocab_file` then you need to add them here at the beginning.